File: src/main/java/com/example/demo/entites/PlaneEntity.java
package com.example.demo.entites;

import com.example.demo.enums.PlaneEnum;

public class PlaneEntity {
    private String throttle;
    private int rudder;
    private int elevator;
    private int ailerionRight;
    private int ailerionLeft;
    public String getThrottle() {
        return throttle;
    }

    public void setThrottle(String throttle) {
        this.throttle = throttle;
    }

    public double getRudder() {
        return rudder;
    }

    public String getStringRudder() {
        return String.valueOf(rudder);
    }
    public void setRudder(int rudder) {
        this.rudder = rudder;
    }


    public double getElevator() {
        return elevator;
    }
    public String getStringElevator() {
        return String.valueOf(elevator);
    }
    public void setElevator(int elevator) {
        this.elevator = elevator;
    }

    public double getAilerionRight() {
        return ailerionRight;
    }

    public String getStringAilerionRight() {
        return String.valueOf(ailerionRight);
    }
    public void setAilerionRight(int ailerionRight) {
        this.ailerionRight = ailerionRight;
    }

    public double getAilerionLeft() {
        return ailerionLeft;
    }
    public String getStringAilerionLeft() {
        return String.valueOf(ailerionLeft);
    }
    public void setAilerionLeft(int ailerionLeft) {
        this.ailerionLeft = ailerionLeft;
    }



}

File: AtakHanMobile4/app/src/main/java/com/example/atakhanmobile/PlaneEntity.kt
package com.example.atakhanmobile

data class PlaneEntity(
    var throttle: String,
    var rudder: Int,
    var elevator: Int,
    var ailerionRight: Int,
    var ailerionLeft: Int
){
    fun copy(): PlaneEntity {
        return PlaneEntity(throttle, rudder, elevator, ailerionRight, ailerionLeft)
    }

    override fun equals(other: Any?): Boolean {
        if (this === other) return true
        if (other !is PlaneEntity) return false

        if (throttle != other.throttle) return false
        if (rudder != other.rudder) return false
        if (elevator != other.elevator) return false
        if (ailerionRight != other.ailerionRight) return false
        if (ailerionLeft != other.ailerionLeft) return false

        return true
    }

    override fun hashCode(): Int {
        var result = throttle.hashCode()
        result = 31 * result + rudder
        result = 31 * result + elevator
        result = 31 * result + ailerionRight
        result = 31 * result + ailerionLeft
        return result
    }
}